functional_tests/steps/bundles.py

```diff
@@ -67,3 +69,52 @@ def the_selected_datasets_are_displayed(context: Context) -> None:
     context.page.get_by_text(
         "Deaths registered weekly in England and Wales by region (Edition: Example Dataset 3, Ver: 1)"
     ).is_visible()
+
+
+# bundle create amend
+@step('a bundle has been created with name "{bundle_name}" and creator')
+def a_bundle_has_been_created_with_user(context: Context, bundle_name: str) -> None:
+    context.bundle_creator = UserFactory()
+    context.bundle = BundleFactory(name=bundle_name, created_by=context.bundle_creator)
+
+
+@step("the bundle has creator removed")
+def delete_bundle_creator(context: Context) -> None:
+    context.bundle_creator.delete()
+
+
+# bundle goto
+@step("the user goes to the bundle inspect page")
+def go_to_bundle_inspect(context: Context) -> None:
+    context.page.goto(context.base_url + "/admin/bundle/inspect/1/")
```

Comment:
We should be careful with hardcoding urls, probably better to do `reverse('bundle:inspect', args=[context.bundle.pk])`